chart/kubeapps/values.yaml

```diff
@@ -83,50 +107,79 @@ frontend:
       port: 8080
     initialDelaySeconds: 0
     timeoutSeconds: 5
-  # https://github.com/kubeapps/kubeapps/issues/478#issuecomment-422979262
+  ## NGINX containers' resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ##
   resources:
+    ## Default values set based on usage data from running Kubeapps instances
+    ## ref: https://github.com/kubeapps/kubeapps/issues/478#issuecomment-422979262
+    ##
     limits:
       cpu: 250m
       memory: 128Mi
     requests:
       cpu: 25m
       memory: 32Mi
-  nodeSelector: {}
-  tolerations: []
+  ## Affinity for pod assignment
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+  ##
   affinity: {}
+  ## Node labels for pod assignment. Evaluated as a template.
+  ## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+  ##
+  nodeSelector: {}
+  ## Tolerations for pod assignment. Evaluated as a template.
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  ##
+  tolerations: {}
+  ## Use access_token as the Bearer when talking to the k8s api server
+  ## Some K8s distributions such as GKE requires it
+  ##
   proxypassAccessTokenAsBearer: false
 
-# AppRepository Controller is the controller used to manage the repositories to
-# sync. Set apprepository.initialRepos to configure the initial set of
-# repositories to use when first installing Kubeapps.
+## AppRepository Controller is the controller used to manage the repositories to
+## sync. Set apprepository.initialRepos to configure the initial set of
+## repositories to use when first installing Kubeapps.
+##
```

Comment:
why the empty line in these comments?

Reply:
It's a standard we're using on our Helm charts for better readability.